.github/workflows/ci.yml

```diff
@@ -267,6 +267,9 @@ jobs:
           -D skipChecks
           test
           | tee "${BUILD_OUTPUT_FILE_PATH}"
+        env:
+          # creates a state of roughly 4GB
+          STATE_CONTROLLER_PERFORMANCE_TESTS_KEYCOUNT: "10000"
```

Comment:
💭 I was thinking what we actually care about is the end size of the state, like "I want to test the performance for a 10 gig snapshot." Would be cool if we could configure this in that way, instead of the key count since this is a bit abstract and not really clear what the end result is. or what do you think?

Reply:
Yeah, being able to have a target size would be best. I struggled a little with this because RocksDB flushes asynchronously, and the only real way to have a "final" DB size is by closing it :smile: (maybe? Maybe I missed something). But yeah, it'd be nicer to target a size and nothing else :thinking: